### PR TITLE
Update GetApolloClientLinksToken implementation for handling terminal links

### PR DESCRIPTION
**Pull request by KevinGrandon**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -97,10 +99,7 @@ const ApolloClientPlugin = createPlugin({
 
         return forward(operation);
       });
-      const links = [authMiddleware, connectionLink];
-      if (apolloLink) {
-        links.unshift(apolloLink);
-      }
+      const links = getApolloLinks([authMiddleware, connectionLink]);
```
Comment (revskill10) [inline src/index.js]: Need to check `if getApolloLinks && typeof(getApolloLinks) === 'function' ? ... : `

Comment (KevinGrandon) [reply src/index.js]: I am not sure we do, because of the function default argument.
```
getApolloLinks = links => links,
```

Comment (revskill10) [reply src/index.js]: @KevinGrandon Ah, ok. Then everything is fine.

**Comment by KevinGrandon** [comment]
Tagging as a feature, technically breaking, but we haven't released the breaking change yet - so just a feature!

**Pull request merged**